modules/openapi-generator/src/main/resources/typescript-fetch/modelGeneric.mustache

```diff
@@ -20,6 +20,17 @@ import {
 {{/discriminator}}
 {{>modelGenericInterfaces}}
 
+export function instanceOf{{classname}}(value: any): boolean {
```

Comment:
please add documentation to this function

the parameter type could be `object`, right?

Reply:
You're right. Enforcing object in here is definitely more proper!

I also added a short docstring to this function :slightly_smiling_face: .